build-all.bat

```diff
@@ -0,0 +1,4 @@
+mvn -f target-platform/pom.xml clean install "-Dmaven.test.skip=true" &&^
```

Comment:
Is this file new? Is it related to this PR or not?

Reply:
no this is a script file of mine.